gui/executionGraphGui/client/apps/eg/src/app/model/Node.ts

```diff
@@ -11,57 +11,54 @@
 // =========================================================================================
 
 import { Point } from '@eg/graph';
-import * as Long from 'long';
-import { Socket, InputSocket, OutputSocket } from './Socket';
-import * as fromSocket from './Socket';
+import { Socket, OutputSocket, InputSocket, isOutputSocket } from './Socket';
 
-function isLong(value: any): value is Long {
-  return value instanceof Long;
+export interface UIProps {
+  readonly name: string;
+  position: Point;
```

Comment:
should become readonly one day. We're too lazy now because the property is set in the effect directly on the node instead of passing it on to the reducer... If that is too slow we should consider an alternative position attribute per node during movements / animations....

Reply:
Jup, workaround so far. put a comment next to it.